test/jdk/java/awt/print/PrinterJob/PolylinePrintingTest.java

```diff
@@ -42,6 +42,24 @@
 import java.awt.print.PrinterJob;
 
 public class PolylinePrintingTest implements Printable {
+    private static final String INSTRUCTIONS = """
+              You must have a printer available to perform this test.
+              OK the print dialog, and collect the printed page.
+              Passing test : Output should show two identical chevrons.
+              Failing test : The line joins will appear different.
+              """;
+
+    public static void main(String[] args) throws Exception {
+        PassFailJFrame passFailJFrame = PassFailJFrame.builder()
+                .instructions(INSTRUCTIONS)
+                .rows((int) INSTRUCTIONS.lines().count() + 2)
+                .columns(45)
+                .build();
+
+        new PolylinePrintingTest();
+
+        passFailJFrame.awaitAndCheck();
+    }
```

Comment:
This second test may also throw `jtreg.SkippedException` if the printer is not available. Then you can remove this statement from the instructions.

I would also place the test code inside the main method as it's done in the `PageRanges.java` test.

I also think setting the page format and paper isn't necessary to reproduce the original bug [JDK-8041902](https://bugs.openjdk.org/browse/JDK-8041902), so this can be removed from the test, which will make the code small enough to inline into the main method.